src/components/TaskModal/taskModal.js

```diff
@@ -19,6 +20,14 @@ export default {
       dueDate: ''
     }
   },
+  created() {
+    console.log(this.editingTask)
+    if (this.editingTask) {
+      this.title = this.editingTask.title
+      this.description = this.editingTask.description
+      this.dueDate = new Date(this.editingTask.date)
```

Comment:
պետք է ստուգել, եթե this.editingTask.date-ը կա, նոր new Date անել, հակառակ դեպքում ստացվում է Invalid date